src/api/app/models/bs_request_action_delete.rb

```diff
@@ -41,8 +41,8 @@ def render_xml_attributes(node)
   end
 
   def sourcediff(opts = {})
-    raise DiffError, "Project diff isn't implemented yet" unless target_package && target_repository
-    return '' unless target_repository
+    raise DiffError, "Project diff isn't implemented yet" unless target_package || target_repository
+    return '' unless target_package
```

Comment:
without cassettes I have the prroblem in spec/models/bs_request_action/differ/for_source_spec.rb - is this related? If not, it would be good to have one more spec :)

```
     context 'with error' do
+      skip('NOVCR nothing was raised')
       it { expect { subject.perform }.to raise_error(BsRequestAction::DiffError) }
     end

Reply:
No, this seems unrelated.